pyproject.toml

```diff
@@ -16,7 +16,8 @@ dependencies = [
   "multidict",
   "jinja2",
   "pyhamcrest",
-  "pytest"
+  "pytest",
+  "mastodon.py"
```

Comment:
Be sure you install the mastodon.py package to use this Node.

Reply:
I did not know such a thing existed, but how convenient!